angular.json

```diff
@@ -32,14 +40,20 @@
             ],
             "scripts": [
               "node_modules/post-robot/dist/post-robot.min.js"
-            ]
+            ],
+            "aot": false,
+            "vendorChunk": true,
+            "extractLicenses": false,
+            "buildOptimizer": false,
+            "sourceMap": true,
+            "optimization": false,
+            "namedChunks": true
           },
           "configurations": {
```

Comment:
Can't comment on the right line because it wasn't changed...but under the staging/local configurations, I would recommend adding
 "sourceMap": true, "optimization": false,
at the same level as the fileReplacements property. From what I've seen this allows you to get the source maps and like properly debug